README.md

```diff
@@ -203,6 +203,6 @@ Copyright (c) 2020 SAP SE or an SAP affiliate company.
 
 Licensed under the **Apache License, Version 2.0** (the "License"); you may not use this file except in compliance with the License.
 
-You may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0.
+You may obtain a copy of the License at [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
```suggestion
You may obtain a copy of the License at <https://www.apache.org/licenses/LICENSE-2.0>.
```

See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Thanks, that looked odd 😉